test/decl/operator/power_operator_imported.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck -I %S/Inputs/custom-modules -I %t %s -verify
+
+// REQUIRES: objc_interop
+
+import PowerOperator
+
+func testInvalidPowerOperationOutOfScope() {
```

Comment:
```suggestion
func testNonexistentPowerOperatorWithPowFunctionInScope() {
```

How about something like that?

Reply:
Thanks! I modified all function names as you suggested! 👍